domehunter/dome_control.py

```diff
@@ -282,27 +287,7 @@ def abort(self):
         # will receive no voltage even if the relay is in the open position?
         self._stop_moving()
 
-    def getAz(self):
-        """
-        Return current Azimuth of the Dome.
-
-        Returns
-        -------
-        float
-            The Dome azimuth in degrees.
-
-        """
-        # TODO: Now that dome_az is a property calculated from encoder_count
-        # it is no longer initialised as None, so instead probably need some
-        # other form of error/exception handling here (if encoder_count is
-        # None?)
-        if self.dome_az is None:
-            print("Cannot return Azimuth as Dome is not yet calibrated.\
-                   Run calibration loop")
-            return self.dome_az
-        return self.dome_az.degree
```

Comment:
Removing `getAz` from `Dome` class is OK, but need to change the corresponding method in the dome server class to access the `.dome_az` property instead. That method is necessary because `getAz` is part of the X2 Dome API and TheSkyX uses it.

Reply:
Will check on this, thanks.